client/src/api/common/data.ts

```diff
@@ -147,3 +147,18 @@ export function startLogging() {
 export function stopLogging() {
   emit('stop-das-recording');
 }
+
+/**
+ * Start Boosting
+ */
+export function startBoosting(){
```

Comment:
This is probably a nit pick, but how about calling this `startBoost` instead? We don't currently have 'boosting' as a terminology so it may confuse others looking at this code.

Reply:
ahahha sure!

Reply:
Also (sorry for picking things late) but just realised we may need to update the tittle that goes with the start/stop button to say something like `Start BOOST + DAS Recording`:
<img width="570" alt="Screen Shot 2021-07-19 at 3 05 55 pm" src="https://user-images.githubusercontent.com/63642262/126105964-2132f0d6-e74d-4d41-89c4-df8d4fe7d491.png">